bibliopixel/drivers/SPI/APA102.py

```diff
@@ -1,24 +1,43 @@
 from ... import gamma
 from .. channel_order import ChannelOrder
+from ... import data_maker
 from . base import SPIBase
 
 
 class APA102(SPIBase):
-    """Driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+    """Driver for APA102/SK9822 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
     def __init__(self, num, gamma=gamma.APA102, **kwargs):
         super().__init__(num, gamma=gamma, **kwargs)
+        maker = kwargs.get('maker', data_maker.MAKER)
 
-        # APA102 requires latch bytes at the end
-        self._latch_bytes = [0xFF, 0, 0, 0] * ((num // 64) + 1)
+        # APA102/SK9822 requires latch bytes at the end)
+        # Many thanks to this article for combined APA102/SK9822 protocol
+        # https://cpldcpu.com/2016/12/13/sk9822-a-clone-of-the-apa102/
+        self._start_frame = 4  # start frame is [0, 0, 0, 0]
```

Comment:
on APA102 per the spec they are actually different for start and end. On SK9822 they are also not really the same either but the result of making a protocol that tricks both into work is that they are the same. Which is why I still kept them different.